datanommer.commands/tests/test_commands.py

```diff
@@ -517,3 +517,12 @@ def test_latest(datanommer_models, mock_config, mock_init):
     assert json_object[1]["git"]["body"] == {"Message 3": "Message 3"}
     assert json_object[0]["fas"]["body"] == {"Message 2": "Message 2"}
     assert len(json_object) == 2
+
+
+def test_refresh_view(datanommer_models):
+    """Test the refresh_view command."""
+
+    runner = CliRunner()
+    result = runner.invoke(datanommer.commands.refresh_view, [])
+    assert result.exit_code == 0, result.output
```

Comment:
It would be better to check that the function has actually been called:

```suggestion
def test_refresh_view(datanommer_models, mock_config, mocker):
    """Test the refresh_view command."""

    runner = CliRunner()
    mock_refresh = mocker.patch("datanommer.commands.refresh_recent_topics")
    result = runner.invoke(datanommer.commands.refresh_view, [])
    assert result.exit_code == 0, result.output
    mock_refresh.assert_called_once_with(m.session)
```

Reply:
I tested the function run inside the VM and it passed the test. Not sure why failed in CI - may be due to some config not setup in CI.

I'll make this change.